ui/app/models/node.js

```diff
@@ -1,4 +1,3 @@
-import { computed } from '@ember/object';
 import { alias, and, equal } from '@ember/object/computed';
 import DS from 'ember-data';
 const { attr } = DS;
```

Comment:
Unrelated, but you ought to be able to use modules for Ember Data as well now.

```js
import Model from 'ember-data/model';
import attr from 'ember-data/attr';
```

Reply:
Huh I guess the codemod doesn't do that?